node-mongodb-high-availability/azuredeploy-parameters.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
this can be renamed now to azuredeploy.parameters.json. We fixed the bug in the bot.